### PR TITLE
new 

### PR DESCRIPTION
**Pull request by dbisstudent**

### CODE REVIEW

File: test/Tereshchenko_Igor/laboratory1.sql [hunk 1]
```diff
@@ -1,2 +1,158 @@
--- LABORATORY WORK 1
--- BY Tereshchenko_Igor
```
Comment (igortereshchenko) [inline test/Tereshchenko_Igor/laboratory1.sql:2]: Delete 
`--- LABORATORY WORK 1
--- BY Tereshchenko_Igor`

File: test/Tereshchenko_Igor/laboratory1.sql [hunk 2]
```diff
@@ -1,2 +1,158 @@
...
+TEMPORARY TABLESPACE "TEMP";
+
+-- QUOTAS
+ALTER USER tereshchenko QUOTA 100M ON USERS;
+
+-- ROLES
+GRANT "CONNECT" TO tereshchenko ;
+
+-- SYSTEM PRIVILEGES
+GRANT CREATE TABLE TO tereshchenko ;
```
Comment (igortereshchenko) [inline test/Tereshchenko_Igor/laboratory1.sql]: `TABLE TO tereshchenko` =>`ANY TABLE TO tereshchenko`

**Review by igortereshchenko — CHANGES_REQUESTED** [review]

File: test/Tereshchenko_Igor/laboratory1.sql (hunk 1 above)
Comment (dbisstudent) [reply test/Tereshchenko_Igor/laboratory1.sql:2]: Fixed

File: test/Tereshchenko_Igor/laboratory1.sql (hunk 2 above)
Comment (dbisstudent) [reply test/Tereshchenko_Igor/laboratory1.sql]: Fixed

**Review by igortereshchenko — APPROVED** [review]

**Pull request closed without merging**